LensmaniaClient/wwwroot/css/app.css

```
@@ -1,23 +1,65 @@
+/* Variables */
+:root {
+    --color-dark-purple: #563EA1;
+    --color-yellow: #FAF9BF;
+    --color-light-purple: #BDB3DD;
+    --color-lighter-purple: #E6E3F2;
+    --color-pink: #F4A6B9;
+
+    --font-main: 'Monserrat', sans-serif, Arial, sans-serif;
```

Stylelint (17.6.0)

[error] Expected "Arial" to be "arial" (value-keyword-case)

(value-keyword-case)